config/environments/development.rb

```diff
@@ -69,15 +72,16 @@
   config.jwt_secret_key = 'development'
 
   config.events = {
-    enabled: false,
+    enabled: true,
```

Comment:
I'll switch this back to false, to stop WO refusing to start if you aren't running RabbitMQ